internal/output/format.go

```diff
@@ -23,12 +26,46 @@ func formatCost(currency string, d *decimal.Decimal) string {
 	if d == nil {
 		return "-"
 	}
+
+	if os.Getenv("INFRACOST_CURRENCY_FORMAT") != ""{
+        return customFormatCost(d)
```

Comment:
Instead of returning a fixed format, setting the custom format should adjust the formatter that is used by the other formatting functions.  (The reason is because this module is used to format prices (which can be very small) and costs which can be large so there isn't really a one-size-fits-all format.)

Maybe the best way to do this is at the start of the output.FormatOutput function use the `money.AddCurrency` to update the formatter for the currently selected currency.

Reply:
I have amended it accordingly, please have a look